### PR TITLE
implement a loterry

### PR DESCRIPTION
**Pull request by aksoli666**
for three balls

### CODE REVIEW

File: src/main/java/core/basesyntax/ColorSupplier.java [hunk 1]
```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public static Colors getRandomColor() {
+        int index = new Random().nextInt(Colors.values().length);
```
Comment (sokimaaa) [inline src/main/java/core/basesyntax/ColorSupplier.java:7]: Think about which variables should be local in the method and which should be class-level

see common mistakes https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery

File: src/main/java/core/basesyntax/Lottery.java [hunk 2]
```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        Random random = new Random();
```
Comment (sokimaaa) [inline src/main/java/core/basesyntax/Lottery.java:7]: Think about which variables should be local in the method and which should be class-level


see common mistakes https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery

File: src/main/java/core/basesyntax/Lottery.java [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+        int numberR = random.nextInt(100);
+        String colorR = ColorSupplier.getRandomColor().toString();
+        return new Ball(colorR, numberR);
```
Comment (sokimaaa) [inline src/main/java/core/basesyntax/Lottery.java:10]: That weird local var name.

I would propose to rename, e.g. `colorR` -> `randomColor` or `color`

File: src/main/java/core/basesyntax/Lottery.java (hunk 2 above)
Comment (aksoli666) [reply src/main/java/core/basesyntax/Lottery.java:14]: do you mean that 
private final Random random;

File: src/main/java/core/basesyntax/Application.java [hunk 4]
```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        for (int i = 0; i < 3; i++) {
+            Ball ball;
+            ball = lottery.getRandomBall();
```
Comment (Rommelua) [inline src/main/java/core/basesyntax/Application.java:8]:
```suggestion
            Ball ball = lottery.getRandomBall();
```

File: src/main/java/core/basesyntax/Colors.java [hunk 5]
```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+public enum Colors {
```
Comment (Rommelua) [inline src/main/java/core/basesyntax/Colors.java:3]:
```suggestion
public enum Color {
```

**Review by Rommelua — CHANGES_REQUESTED** [review]

**Review by Rommelua — APPROVED** [review]

**Pull request closed without merging**